docs/langref/relay_intro.rst

```diff
@@ -0,0 +1,25 @@
+=====================
+Introduction to Relay
+=====================
```

Comment:
we can directly move this content to a section index.rst

Reply:
I will make that change soon